examples/kotlin/package.json

```diff
@@ -7,6 +7,8 @@
         "@bazel/terser": "^1.5.0",
         "domino": "^2.1.3",
         "http-server": "^0.11.1",
+        "jasmine": "^3.5.0",
+        "jasmine-core": "^3.5.0",
```

Comment:
doesn't make sense to me why end users need to know about the `jasmine-core` package. If it isn't in jasmine's docs, you shouldn't need under Bazel either.